dds/DCPS/LogAddr.h

```diff
@@ -0,0 +1,32 @@
+#ifndef OPENDDS_DCPS_LOGADDR_H
+#define OPENDDS_DCPS_LOGADDR_H
+
+#include "dcps_export.h"
+#include "PoolAllocator.h"
+
+#include <dds/Versioned_Namespace.h>
+
+#include <ace/INET_Addr.h>
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace OpenDDS {
+namespace DCPS {
+
+class OpenDDS_Dcps_Export LogAddr
+{
+public:
+  enum Fmt {IP, IP_PORT, HOST, HOST_PORT, IP_PORT_HOST};
```

Comment:
I'm a little nervous about using things like `IP` and `HOST` because they might collide with macros. How about using capitalized camel case like we do [here](https://github.com/objectcomputing/OpenDDS/blob/cd813f4e19416c05acec937b3de784663fb1e0ad/dds/DCPS/Serializer.h#L693-L695)?

Reply:
Good catch, since these symbols are so common.